tests/parser/file_not_found.eclair

```diff
@@ -0,0 +1,8 @@
+// RUN: mkdir -p %t && sed s@TEST_DIR@%t@g %s > %t/input.test
+// RUN: split-file %t/input.test %t
+
+// RUN: %eclair compile foo.eclair > %t/actual_output
```

Comment:
I had to do `>` here instead of `2>` as I could not find a suitable method to print to `stderr`

Reply:
I fixed that for you, but basically you can use `hPutStrLn`, which is like `putStrLn`, but takes a `Handle` (stdout, stderr, other file descriptors...) where it writes the line too.